src/utils/functions.js

```diff
@@ -1,6 +1,7 @@
+// eslint-disable-next-line no-unused-vars
 const Discord = require("discord.js")
```

Comment:
If this is not used, then why does the file require it?

Reply:
JSdoc annotation for the function. I guess I'll  remove it